### ISSUE TITLE
Mentor/Mentor Managers About Page-Admin-App - Sprint 3

### ISSUE THREAD

**Issue by peculiaruc** [ALCOpenSource/Mentor-Management-System-Team-6#24]
Description:

- Build the Screen using the style guide.
- Integrate Screen with the Backend

![image](https://user-images.githubusercontent.com/35475543/222643628-4a3af352-f63f-4339-9c3e-0a46041e0b31.png)

**Comment by Munalula-Sikazwe** [comment]
@askinss 
Proceeding to work on this sprint 3 task as it is a blocker for a sprint 2 task.

**Comment by heysolomon** [comment]
@askinss I want to work on this issue

**Comment by askinss** [comment]
@heysolomon I believe the backend is already available. Can integration happen asap?

**Comment by heysolomon** [comment]
> @heysolomon I believe the backend is already available. Can integration happen asap?

Yes it can, I found endpoints related to this issue on the documentation. I will work on it.
